docs/install/network_options.md

```diff
@@ -42,7 +42,9 @@ The next tabs inform how to deploy each CNI plugin and override the default opti
         flannel:
           backend: "wireguard"
     ```
-    
+
+    After that, please restart the canal daemonset to use the newer config.    
```

Comment:
Can we provide an example command to accomplish this? Users may not know how to trigger a restart of all the daemonset pods.

Reply:
done!